cli/template/extras/prisma/schema/base-planetscale.prisma

```diff
@@ -0,0 +1,21 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+    provider        = "prisma-client-js"
+    previewFeatures = ["driverAdapters"]
+}
+
+datasource db {
+    provider = "sqlite"
```

Comment:
Should this not be `mysql` since this is a planetscale prisma schema?

Also probably needs `relationMode = "prisma"` in this section too.